7-loops/For Each

```diff
@@ -0,0 +1,49 @@
+New Feature added in ES 6 
```

Comment:
the file should be a .js file, it's not the case, please rename it `for-each-loop.js`